package.json

```diff
@@ -132,8 +132,8 @@
     "ovh-common-style": "ovh-ux/ovh-common-style#^3.2.2",
     "ovh-jquery-ui-draggable-ng": "ovh-ux/ovh-jquery-ui-draggable-ng#^0.0.5",
     "ovh-manager-webfont": "ovh-ux/ovh-manager-webfont#^1.0.1",
-    "ovh-ui-angular": "~2.17.1",
-    "ovh-ui-kit": "~2.17.1",
+    "ovh-ui-angular": "^2.17.1",
```

Comment:
Please don't, there are breaking changes between minor version in `ovh-ui-angular`

Reply:
Without updating it, my code mostly breaks and stuff goes wrong all around the place :/

Reply:
It may break things all over the manager though

Reply:
Better make them break and fix'em?